packages/ui/src/components/Container.tsx

```diff
@@ -1 +1,12 @@
-export { Container, type ContainerProps } from 'theme-ui';
+import * as React from 'react';
+import { type ContainerProps, Container as ContainerUi } from 'theme-ui';
+
+export type { ContainerProps };
+
+export const Container = React.forwardRef<HTMLDivElement, ContainerProps>(
+  (props, ref) => {
+    return <ContainerUi ref={ref} {...props} />;
+  }
+);
+
+Container.displayName = 'Container';
```

Comment:
Why, did you decide to make this component this way?

Reply:
To avoid this Typescript error https://github.com/microsoft/TypeScript/issues/42873!